integrationTests/src/test/java/org/sonarqube/gradle/AbstractGradleIT.java

```diff
@@ -129,7 +129,7 @@ protected RunResult runGradlewWithEnvQuietly(String project, String exeRelativeP
       .directory(exeDir)
       .redirectOutput(outputFile)
       .redirectErrorStream(true);
-    pb.environment().put("GRADLE_OPTS", "-Xmx1024m");
+    pb.environment().put("GRADLE_OPTS", "-Xmx1024m --add-opens=java.prefs/java.util.prefs=ALL-UNNAMED");
```

Comment:
Does it mean that to run the scanner with java 17 we need to add this JVM parameter?
What fails if this is not passed?

Reply:
It had this issue with Java 17(and gradle 7.3+ doesn't fix it): https://github.com/gradle/gradle/issues/18508